.circleci/images/primary/Dockerfile-2.3.8

```diff
@@ -57,19 +131,6 @@ RUN DOCKERIZE_URL="https://github.com/powerman/dockerize/releases/download/v0.17
   && chmod +x /usr/local/bin/dockerize \
   && dockerize --version
 
-# Install RubyGems
-RUN mkdir -p "$GEM_HOME" && chmod -R 777 "$GEM_HOME"
-
-# Install RubyGems and Bundler
-# NOTE: Rubygems 3.0.6 is the last version that seems to work fine in this image AND not drag bundler 2 along.
-# Later versions are either broken or they force the use of bundler 2, which we can't use because some of our
-# dependencies (e.g. rails 3.0.20) don't like it.
-RUN gem update --system '3.0.6'
-# Ruby 2.3 can support Bundler 2+
-# But hold back to < 2 for now, because some dependencies require it.
-RUN gem install bundler -v '1.17.3'
-ENV BUNDLE_SILENCE_ROOT_WARNING 1
-
```

Comment:
I noticed that, unlike the other images, we're not installing a specific bundler version. This should probably be fine, but let's keep an eye out for it :)